.github/workflows/docker-build-push.yml

```diff
@@ -12,10 +12,20 @@ on:
         required: false
         type: string
         default: ./docker/Dockerfile
+    secrets:
+      DOCKERHUB_USERNAME:
+        description: The Docker Hub username.
+        required: true
+      DOCKERHUB_TOKEN:
+        description: A token to push images to the Docker Hub repository.
+        required: true
+      BUILD_ARGS:
```

Comment:
what would such build arguments be? sounds more like `inputs` to me at the first glance